packages/kit/types/hooks.d.ts

```diff
@@ -16,19 +16,19 @@ export interface ServerResponse {
 	body?: StrictBody;
 }
 
-export interface GetSession<Locals = Record<string, any>, Session = any> {
-	(request: ServerRequest<Locals>): MaybePromise<Session>;
+export interface GetSession<Locals = Record<string, any>, Body = unknown, Session = any> {
```

Comment:
Inserting the argument in the middle will be a breaking change, but it does make sense considering the usage order and keeps it consistent with the order from other types like `Load`. Let's make sure to also update the docs accordingly so users can crosscheck properly.

Reply:
Alternatively, one could use variadic tuples if BC should be kept: https://www.typescriptlang.org/docs/handbook/release-notes/typescript-4-0.html

This would result in more work and bloat in the implementation, so I would personally vote against it. Just wanted to point out that it would be possible.

Reply:
@ignatiusmb I updated the docs to reflect the changes.
@benmccann I agree with you there, it's cool that it's possible but I don't think it's worth the bloat.